app/server/logging/jsondb/src/main/java/io/syndesis/server/logging/jsondb/controller/PodLogMonitor.java

```diff
@@ -198,7 +198,7 @@ InflightData getInflightData(String exchangeId, String logts) throws IOException
     }
 
     @SuppressWarnings({"PMD.CyclomaticComplexity", "PMD.ExcessiveMethodLength", "PMD.NPathComplexity", ""})
-    private void processLine(String line) throws IOException {
+    void processLine(String line) throws IOException {
```

Comment:
where else is processLine used ?

Reply:
I think that's a leftover from the test I deleted where I needed to override the method.